Core/Program.cs

```diff
@@ -40,9 +40,11 @@ public static int Main (string[] args)
     {
       app.Run(args);
     }
-    catch (UserDoesNotWantToContinueException u)
+    catch (UserInteractionException e)
     {
-      Console.WriteLine(u.Message);
+      Console.WriteLine(e.Message);
+      var log = Log.ForContext(e.TargetSite!.DeclaringType);
+      log.Error(e.Message);
```

Comment:
```suggestion
      log.Error(e, "User cancled the operation or cannot continue with release due to inconsistencies in the repository or JIRA.");
```

In the log file, the stacktrace is helpful in case we do need it.